src/AppBundle/Command/AddUserCommand.php

```diff
@@ -61,8 +61,18 @@ protected function configure()
     }
 
     /**
-     * This method is executed before initialize() and execute(). Its purpose is
-     * to check if some of the options/arguments are missing and interactively
+     * This method is executed before the interact() and the execute() methods.
+     * It's main purpose is to initialize the variables used in the rest of the
+     * command methods.
```

Comment:
I would add a note saying that the input has not been validated yet here (as this is done only after ``interact``, so the input should be used with great caution here (or not used at all)